Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -294,26 +359,26 @@ export default function widgetBehavior(publicAPI, model) {
       model._apiSpecificRenderWindow
     );
     worldCoords = publicAPI.getBoundedCenter(worldCoords);
-    model.activeState.setCenter(worldCoords);
-    updateState(model.widgetState);
+    model.widgetState.setCenter(worldCoords);
+    updateState(
+      model.widgetState,
+      model._factory.getDisplayScaleParams(),
+      model._factory.getRotationHandlePosition()
+    );
   };
 
   publicAPI[InteractionMethodsName.RotateLine] = (calldata) => {
-    const activeLine = model.widgetState.getActiveLineState();
+    const activeLineHandle = publicAPI.getActiveLineHandle();
     const planeNormal = model.planeManipulator.getWidgetNormal();
     const worldCoords = model.planeManipulator.handleEvent(
       calldata,
       model._apiSpecificRenderWindow
     );
 
     const center = model.widgetState.getCenter();
-    const previousLineDirection = vtkMath.subtract(
-      activeLine.getPoint1(),
-      activeLine.getPoint2(),
-      []
-    );
+    const previousLineDirection = activeLineHandle.getDirection();
     vtkMath.normalize(previousLineDirection);
-    if (model.widgetState.getActiveRotationPointName() === 'point1') {
+    if (publicAPI.getActiveRotationPointName() === 'point0') {
```

Comment:
It may be better to use descriptive comments for these magic string names, if applicable.

Reply:
I just added some comments in `getActiveRotationPointName()`